### PR TITLE
Add s3 remote logging example

### PR DESCRIPTION
**Pull request by krisgeus**

### CODE REVIEW

File: envs/airflow-s3-logging/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+  airflow:
+    image: docker-whirl-airflow:py-${PYTHON_VERSION}-local
```
Comment (NielsZeilemaker) [inline envs/airflow-s3-logging/docker-compose.yml]: You can actually build from compose, eg:
```
        build:
          context: .
          dockerfile: Dockerfile_airflow
          args:
            AIRFLOW_VERSION: master
            AIRFLOW_EXTRAS: async,celery,crypto,jdbc,hdfs,hive,azure,gcp_api,emr,password,postgres,slack,ssh,atlas
```
You can then rebuild using `docker-compose up --build`

File: envs/airflow-s3-logging/whirl.setup.d/01_add_connection_s3.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+echo -e "$AWS_ACCESS_KEY_ID\n$AWS_SECRET_ACCESS_KEY\n\n" | aws configure
+aws configure set plugins.endpoint awscli_plugin_endpoint
+aws configure set default.s3.endpoint_url http://${AWS_SERVER}:${AWS_PORT}
+aws configure set default.s3api.endpoint_url http://${AWS_SERVER}:${AWS_PORT}
+
+echo "================================"
+echo "== Create S3 Bucket ==========="
+echo "================================"
+while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://${AWS_SERVER}:${PORT_WEB_UI})" != "200" ]]; do
+  echo "Waiting for s3server to come up..."
```
Comment (bjgbeelen) [inline envs/airflow-s3-logging/whirl.setup.d/01_add_connection_s3.sh]: small comment, perhaps replace `s3server` with `${AWS_SERVER}`

File: envs/airflow-s3-logging/whirl.setup.d/01_add_connection_s3.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  sleep 2;
+done
+
+echo "creating bucket"
+S3_SERVER_IP=$(dig +short s3server)
```
Comment (bjgbeelen) [inline envs/airflow-s3-logging/whirl.setup.d/01_add_connection_s3.sh]: same here

**Comment by bjgbeelen** [comment]
Perhaps write a small section about this example (maybe just its mere existence by starting the examples section with a list of available examples pointing to the example descriptions if there is one).

**Pull request merged**